script.js

```diff
@@ -0,0 +1,24 @@
+let selectOperator = document.querySelector('.operator')
+let a = document.querySelector('.firstInput')
+let b = document.querySelector('.secondInput')
+let result = document.querySelector('.result')
+let btn = document.querySelector('.btn')
+
+function operation() {
+    let newOperator = (selectOperator.value).toLowerCase()
+    let newA = parseInt(a.value)
+    let newB = parseInt(b.value)
+    if (newOperator == 'a') {
```

Comment:
You need just one operation the repetition is not necessary

Reply:
okay.